test/HOC/valid.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+
+import handler from 'react-freeform/HOC/handler';
+import valid from 'react-freeform/HOC/valid';
+
+// Tests for components that this extends or uses
+import './handler';
+
+const defaultValues = { a: true };
+class Test extends React.Component { getDefaults() { return defaultValues; } }
```

Comment:
Not sure if you planned on using `defaultValues` somewhere else but a variable is not needed here